ui/app/AppLayouts/Chat/views/ChatView.qml

```diff
@@ -160,17 +160,33 @@ StatusSectionLayout {
             return false
         }
 
-        let chatContentModule = root.rootStore.currentChatContentModule()
-        if (!chatContentModule) {
+        if (!root.chatContentModule) {
             return false
         }
         // Check if user list is available as an option for particular chat content module
-        return chatContentModule.chatDetails.isUsersListAvailable
+        return root.chatContentModule.chatDetails.isUsersListAvailable
     }
 
     rightPanel: Component {
         id: userListComponent
         UserListPanel {
+            id: userListPanel
+            readonly property bool isFullCommunityList: !root.chatContentModule || !root.chatContentModule.chatDetails || !root.chatContentModule.chatDetails.requiresPermissions
```

Comment:
both `isFullCommunityList` and the `Connections` below have a dependency on the `onChatContentModuleChanged` signal. The implementation assumes that `isFullCommunityList` will always receive the signal first and its value is updated before handling the connection signal.
I'd redo this a little bit to remove any dependency on the signal order.